src/main/java/io/supertokens/config/CoreConfig.java

```diff
@@ -590,6 +631,28 @@ void normalizeAndValidate(Main main, boolean includeConfigFilePath) throws Inval
             }
         }
 
+        try {
+            String[] allowedPasswordHashingAlgos = CoreConfig.class.getDeclaredField("password_hashing_alg")
```

Comment:
you can make the generic, iterate thru all properties and check for this annotation. we might miss adding this logic if we add new enum properties.

Reply:
Done! Although I wasn't able to figure out a way to test it. Can you check if it looks good or let me know if you know to check that it works correctly?